### ISSUE TITLE
Suggestion: Setting to change color of greenwall generated guild messages

### ISSUE THREAD

**Issue by pbjtime** [AIE-Guild/GreenWall#196]
During high traffic chatter, if there are different conversations it is very hard to follow and differentiate. Would be nice to have the option to change the color of text of greenwall generated guild messages

**Comment by mrogaski** [comment]
Greenwall doesn't perform any chat formatting, it sends the messages to the system to display.  Conditional formatting of the chat messages displayed would have to be added in a addon like Prat or ElvUI.

**Comment by pbjtime** [comment]
I see. In that case i suppose this suggestion is moot. Thanks anyways.
